logs/logs.go

```diff
@@ -33,9 +35,60 @@ type LogLine struct {
 	Labels        map[string]string `json:"labels,omitempty"`
 }
 
+func (t LogLine) GetDedupField(field string) string {
+	switch field {
+	case "message":
+		return fmt.Sprintf("msg::%s", t.Message) // TODO: return tokenized message
+	case "hash":
+		return fmt.Sprintf("hash::%s", t.Hash)
+	case "severity":
+		return fmt.Sprintf("severity::%s", t.Severity)
+	case "source":
+		return fmt.Sprintf("source::%s", t.Source)
+	case "host":
+		return fmt.Sprintf("host::%s", t.Host)
+	case "firstObserved":
```

Comment:
first,lastobserver and count are outputs of the dedupe process - it however might make sense to specify the dedupe window e.g. round-down to the last (second, minute, 10m, 15m, 30m, hour,4h, day)

Reply:
should we do

```yaml
dedupe:
  window: 1h
  fields:
    - source
    - hash
```

instead of 

```yaml
dedupe:
    - source
    - hash
```